components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -1633,6 +1633,99 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+#########################################################
+# The "Configure Monetization" Processing Function resource API
+#########################################################
+  /apis/{apiId}/monetize:
+#---------------------------------------
+# Configure monetization for a given API
+#---------------------------------------
+    post:
+      x-scope: apim:api_publish
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -H \"Content-Type: application/json\" -X POST -d @data.json \"https://127.0.0.1:9443/api/am/publisher/v1.0/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/monetize\""
+      x-wso2-request: "POST https://127.0.0.1:9443/api/am/publisher/v1.0/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/monetize\nAuthorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\nContent-Type: application/json\n{\r\n   \"enabled\":\"true\",\r\n   \"properties\":{\r\n      \"vendor\":\"test\",\r\n      \"type\":\"postpaid\"\r\n   }\r\n}"
+      x-wso2-response: "HTTP/1.1 200 OK\nContent-Type: application/json\n{\r\n   \"enabled\":\"true\",\r\n   \"properties\":{\r\n      \"vendor\":\"test\",\r\n      \"type\":\"postpaid\"\r\n   }\r\n}"
+      summary: Configure monetization for a given API
+      description: |
+        This operation can be used to configure monetization for a given API.
+      parameters:
+        - $ref: '#/parameters/apiId'
+        - in: body
+          name: body
+          description: |
+            Document object that needs to be added
```

Comment:
Is this description correct?

Reply:
Fixed : https://github.com/wso2/carbon-apimgt/pull/6448/commits/814a97b190129c385e21bd3c597b4ef58cc72996